tests/installation/bootloader_s390.pm

```diff
@@ -90,8 +90,9 @@ sub prepare_parmfile {
     }
     else {
         if (get_var('AGAMA')) {
+            my $flavor = get_var('FLAVOR');
```

Comment:
use `get_var` directly below, it is better to avoid to create some many variables, it is readable enough.